applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -97,6 +97,14 @@
                         )
        ).
 
+-define(DEFAULT_FORWARD_TYPE(AccountId)
+       ,kapps_account_config:get_global(AccountId
```

Comment:
@icehess this should not be an account based config, as it requires the system to support it.  Lets change this to just kapps_config please ;)